src/TestSuite/IntegrationTestTrait.php

```diff
@@ -478,7 +472,6 @@ public function head($url): void
      *
      * @param string|array $url The URL to request.
      * @return void
-     * @throws \PHPUnit\Framework\Error\Error|\Throwable
```

Comment:
But this is not really correct, see my intial Comments on the original pr.

Reply:
As soon as you disableErrorHandlerMiddleware() you can actually expect the \Throwable at least.